docs/api-reference/carto/carto-layer.md

```diff
@@ -110,6 +154,14 @@ Optional. A string pointing to a unique attribute at the result of the query. A
 
 Optional. Overrides the configuration to connect with CARTO. Check the parameters [here](overview#carto-credentials).
 
+##### `aggregationExp` (String, optional)
+
+Optional. Aggregration SQL expression. Only used for spatial index datasets.
```

Comment:
Typo on "aggregration". Also repeated below for aggregationResLevel.

Reply:
Good catch! :)